doc/Developing/os/Health-Information.md

```diff
@@ -139,9 +139,20 @@ well as pre_cached data. The index ($index) and the sub_indexes (in
 case the oid is indexed multiple times) are also available: if
 $index="1.20", then $subindex0="1" and $subindex1="20".
 
+##### Handling mismatched indexes
+
 When referencing an oid in another table the full index will be used to match the other table.
-If this is undesirable, you may use a single sub index by appending the sub index after a colon to
-the variable name.  Example `{{ $ifName:2 }}`
+If this is undesirable, there are ways to affect the indexes that will be searched for data.
+This may be used in either simple template format (as shown below) or the full value of the field.
+
+Hardcoded index: `{{ $ifName.5 }}`
```

Comment:
Is it worth showing example values these are being run against so people can understand how they can be used?

Reply:
Probably an entirely separate page.